plugins/powerpaste.md

```diff
@@ -186,32 +186,94 @@ tinymce.init({
 
 ## Advanced Config Options
 
-### Post filter callback
+### Pre- and post- filtering callbacks
 
-Developers can add customer filtering after **PowerPaste** filters are run using the post filter callback. This can be added as an init option or at runtime by adding the event listener.
+Developers can add custom filtering before and after **PowerPaste's** filters are run using the pre- and post- filtering callbacks. These can be added as init options or at runtime using event listeners.
```

Comment:
Should the `'s` be outside the bold markup as it's not part of the name?

Also the `pre-` and `post-` looks pretty weird here. Should it either be `pre-filtering and post-filtering` or just `pre and post filtering`?

Reply:
@tylerkelly13 ?

Reply:
First point, Fair.
I ignored that 'cos I liked it. If changing make it `pre-filtering and post-filtering`